pandas/tests/plotting/test_series.py

```diff
@@ -729,6 +729,26 @@ def test_dup_datetime_index_plot(self):
         s = Series(values, index=index)
         _check_plot_works(s.plot)
 
+    def test_errorbar_asymmetrical(self):
+        # GH9536
+        s = Series(np.arange(10), name="x")
+        err = np.random.rand(2, 10)
+
+        ax = s.plot(yerr=err, xerr=err)
+
+        result = np.vstack([i.vertices[:, 1] for i in ax.collections[1].get_paths()])
+        expected = (err.T * np.array([-1, 1])) + list(zip(s, s))
+        tm.assert_almost_equal(result, expected)
```

Comment:
I think this should be `tm.assert_numpy_array_equal`

Reply:
You're right. I had some floating point issues in the beginning, but now the numbers seem to be stable. Changed in https://github.com/pandas-dev/pandas/pull/34514/commits/4f430b3feafc948875783f847c0ba0a1d64692e1